portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -29,68 +29,93 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;  
 
 /** This servet is responsible for the comment data **/
 @WebServlet("/commentServlet")
 public class CommentServlet extends HttpServlet {
   private static final String USERNAME_PARAMETER = "username";
   private static final String REFLECTION_PARAMETER = "reflection";
   private static final String COMMENT_PARAMETER = "Comment";
-  private static final String TIMESTAMP_PARAMETER = "timestamp";
+  private static final String DATE_TIME_PARAMETER = "timestamp";
+  private static final String MAXCOMMENT_PARAMETER = "max-comment";
   private static final String APPLICATION_JSON_PARAMETER = "application/json;";
   private static final String COMMENT_HTML_PARAMETER = "/comments.html";
+  private static final String COMMENT_FORM = "comment-form";
+  private static final String MAX_FORM = "max-form";
+  private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
 
-
-  private String userNameInput;
-  private String reflectionInput;
-  private long idGiven;
-  private long timestampOfComment;  
-
+  private int maxComment = 5; 
   public List<Comment> comments;
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    this.userNameInput = request.getParameter(this.USERNAME_PARAMETER);
-    this.reflectionInput = request.getParameter(this.REFLECTION_PARAMETER);
-    this.timestampOfComment = System.currentTimeMillis();
+    if (request.getParameter(COMMENT_FORM) != null && !request.getParameter(COMMENT_FORM).isEmpty()) {
+        storeEntities(createEntities(request));
+    } else if (request.getParameter(MAX_FORM) != null && !request.getParameter(MAX_FORM).isEmpty()) {
+      setMaxComment(request, request.getParameter(MAXCOMMENT_PARAMETER));
```

Comment:
spacing not consistent here. Use 4 spaces for indentation. See the style guide https://google.github.io/styleguide/javaguide.html